go-controller/pkg/ovn/loadbalancer/lb_cache.go

```diff
@@ -93,6 +100,50 @@ func (c *LBCache) removeVips(toRemove []DeleteVIPEntry) {
 	}
 }
 
+// RemoveSwitches removes the provided toRemoveSwitches which is a list
+// of switches from all the lb's.Switches in the LBCache.
+func (c *LBCache) RemoveSwitches(toRemoveSwitches []string) {
+	c.Lock()
+	defer c.Unlock()
+	for _, sw := range toRemoveSwitches {
+		switches, err := findTableLBs("logical_switch", sw)
+		if err != nil {
+			// best effort, we don't fail
+			klog.Error("Unable to remove load balancers from the switch %s", sw)
+		}
+
+		for switchname, lbuuids := range switches {
+			for _, lbuuid := range lbuuids {
+				if lb, ok := c.existing[lbuuid]; ok {
```

Comment:
any value in emitting a log if it doesn't exist? same for ln 139.

Reply:
nope, its best effort. If its gone already from the cache, we don't care.